openml/datasets/dataset.py

```diff
@@ -697,7 +664,7 @@ def get_data(
         include_ignore_attribute: bool = False,
         dataset_format: str = "dataframe",
     ) -> Tuple[
-        Union[np.ndarray, pd.DataFrame, scipy.sparse.csr_matrix],
+        Union[np.ndarray, pd.DataFrame, pd.SparseDtype],
```

Comment:
This looks wrong. Do we return spares Dtype instead of sparse scipy matrices now?